.azure-pipelines/test-eh-sb-sdk.yml

```diff
@@ -61,40 +80,58 @@ steps:
       runProxy: true
       rootFolder: azure-sdk-for-python
       templateRoot: azure-sdk-for-python
-  - script: |
-      python --version
-      echo "Installing $DownloadArtifactWhl"
-      echo "Full name $(Pipeline.Workspace)/$(DownloadArtifactFolder)/$(DownloadArtifactWhl)"
-      python -m pip install $(Pipeline.Workspace)/$(DownloadArtifactFolder)/$(DownloadArtifactWhl) --force
-      python -m pip install pytest
-      cd $(SdkPath)
-      python -m pip install futures
-      python -m pip install .
-      python -m pip install -r dev_requirements.txt
-      python -m pip list
-      python -m pytest tests -v --doctest-modules $(IgnoreTests) --junitxml=junit/$(TestResultsPath)
-    displayName: 'Run tests'
+  - task: AzurePowerShell@5
+    displayName: Run Tests (AzurePowerShell@5)
     env:
+      SYSTEM_ACCESSTOKEN: $(System.AccessToken)
       AZURE_SUBSCRIPTION_ID: $(azure-subscription-id)
```

Comment:
This should be removed or replaced inline. Is it needed?

Reply:
It's needed in the SB/EH livetests, b/c we create resources using our own fixture/preparers since they're amqp and not http. We have it available here too: https://github.com/Azure/azure-sdk-for-python/blob/61e0fce835ec2640f1c1ef5ac99fc56c118326d7/sdk/eventhub/tests.yml#L29

by inline, do you mean something like `$env:AZURE_SUBSCRIPTION_ID = $account.Id;` below?

Reply:
addressed!

Reply:
looks like it's not actually being set inline? https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4074639&view=logs&j=3264ca38-67bb-54fa-fd91-0bce2e49d546&t=2a681e29-7c4b-5e61-deff-65adf6363d8f&l=11030

Reply:
changing back for now to test credential

Reply:
OOB: the Get-AzContext.* was not being called correctly. Updated, thanks @benbp!